### ISSUE TITLE
re-visit gulliver/lilliput logging (Trac #882)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#876]
gulliver logging can be very "noise" - we need to go through the code and try to sanitize it

Migrated from https://code.icecube.wisc.edu/ticket/882
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "gulliver logging can be very \"noise\" - we need to go through the code and try to sanitize it", 
    "reporter": "claudio.kopper", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "re-visit gulliver/lilliput logging", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-02-23T03:55:00", 
    "milestone": "", 
    "owner": "kkrings", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150810T22:53:37: olivas changed milestone from "" to "IceRec V5"

**Comment by icecube-trac** [comment]
Trac update at 20150831T16:32:41: 
* olivas changed owner from "claudio.kopper" to "olivas"
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20150908T19:48:18: olivas changed status from "assigned" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20150929T14:00:09: 
* kkrings changed owner from "olivas" to "kkrings"
* kkrings changed status from "accepted" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20150929T14:00:16: kkrings changed status from "assigned" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20150930T09:51:04: boersma commented:

> I thought I addressed at least some of this issue with this commit 6 months ago (sorry for the grumpy svn log message):
> http://code.icecube.wisc.edu/projects/icecube/changeset/131565/IceCube
> 
> At the time I was not aware of this ticket, I think, maybe I should have closed it. Unless someone thinks that the modules are still too noisy.
> 
> One major source of noise is Minuit, which is hard to silence with normal means. A brute force solution is to always use the local minuit copy (which was hacked by Claudio to be more silent) instead of the minuit provided by the ROOT libraries. In order to avoid linking trouble when the rest of icetray is built with USE_ROOT=ON, I think it is safest to rename the TMinuit class (e.g. to TMinuit1). I have done this on my laptop, and will create a branch of lilliput some time today (evening time in Europe) so people can have a check if this what we want.

**Comment by icecube-trac** [comment]
Trac update at 20150930T21:08:04: boersma commented:

> the lilliput branch in which we *always* (as opposed to only in `USE_ROOT=OFF` builds) use the local Claudio-hacked logging-friendly version of `TMinuit`:
> http://code.icecube.wisc.edu/svn/projects/lilliput/branches/TMinuit1
>

**Comment by icecube-trac** [comment]
Trac update at 20160114T10:20:24: 
* kkrings commented:

> In changeset 140910:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="140910"
> Merged TMinuit1 branch; closes #876.
> ```


* kkrings changed status from "accepted" to "closed"
* kkrings changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas commented:

> Milestone IceRec V5 deleted


* olivas changed milestone from "IceRec V5" to ""
